src/content/cloud/custom-domains.mdx

```diff
@@ -26,9 +24,36 @@ Perform the following tasks to register your custom domain in your Okteto Cloud
 
 1. Confirm that you own the custom domain name. Need a custom domain name? You can buy one with a domain registration service.
 
-2. [Reach out to support](https://okteto.com/support) to register your custom domain with Okteto Cloud. Please include the domain, your github ID, and the namespace you want to use in your communications.
+2. Update your DNS configuration so your custom domain points to [Okteto DNS target](/docs/cloud/custom-domains/#configure-your-dns).
+
+3. (Only for Okteto Cloud users) [Reach out to support](https://okteto.com/support) to register your custom domain with Okteto Cloud. Please include the domain, your github ID, and the namespace you want to use in your communications. This isn't needed for Scale and Enterpise users.
+
+3. (Only for Scale and Enterprise users) Execute the following commands in your namespace:
 
-3. Update your DNS configuration so your custom domain points to Okteto Cloud's DNS target.
+```
+kubectl label ns ${OKTETO_NAMESPACE} dev.okteto.com/custom-domain=true --overwrite
```

Comment:
It's not clear what OKTETO_NAMESPACE stands for.

Reply:
You could run it in your pipeline if needed, or manually. `OKTETO_NAMESPACE` is the namespace you are configuring with a custom domain